src/components/PopupWithChangeProfile.js

```diff
@@ -0,0 +1,35 @@
+import Popup from "./Popup.js";
+
```

Comment:
`PopupWithChangeProfile.js` is not needed because the `profile` modal should be created with `PopupWithForm`. It's redundant